x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/get_signals_template.ts

```diff
@@ -29,7 +29,7 @@ import aadFieldConversion from './signal_aad_mapping.json';
   incremented by 10 in order to add "room" for the aforementioned patch
   release
 */
-export const SIGNALS_TEMPLATE_VERSION = 55;
+export const SIGNALS_TEMPLATE_VERSION = 56;
```

Comment:
This was already updated this release so it can stay at 55

Reply:
Ah ok. Maybe just note here to other developers that if you need to force the update on your local, upping this does that easy.

Reply:
Do we still not need this? We were running into an issue where the templates weren't updating from the changes in this PR until this value was changed?

Reply:
I think what @marshallmain is saying that it only should be bumped once for the minor release. Locally though, we can bump it to force the re-index.